docs/src/main/sphinx/installation/cli.rst

```diff
@@ -287,3 +292,129 @@ To view debug information, including the stack trace for failures, use the
     at java.base/java.lang.Thread.run(Thread.java:834)
     select count(*) from tpch.tiny.nations
 
+.. _cli-options:
+
+Options
+-------
+
+The following table lists all supported command line options for the Trino CLI
+in alphabetical order:
+
+.. list-table::
+  :widths: 40, 60
+  :header-rows: 1
+
+  * - Option
+    - Description
+  * - ``--access-token``
+    - Access token for :ref:`cli-jwt-auth`.
+  * - ``--catalog``
+    - Default catalog.
+  * - ``--client-info``
+    - Extra information about client making query.
+  * - ``--client-request-timeout``
+    - Client request timeout (default: 2m).
+  * - ``--client-tags``
+    - Client tags.
+  * - ``--debug``
+    - Enable debug information.
+  * - ``--disable-compression``
+    - Disable compression of query results.
+  * - ``--execute``
+    - Execute specified statements and exit.
+  * - ``--external-authentication``
+    - Enable external authentication for :ref:`cli-external-sso-auth`.
+  * - ``--external-authentication-redirect-handler``
+    - External authentication redirect handlers: DESKTOP_OPEN, SYSTEM_OPEN,
+      PRINT, OPEN, ALL (default: ALL).
+  * - ``--extra-credential``
+    - Extra credentials (property can be used multiple times; format is
+      key=value).
+  * - ``-f`` ``--file``
+    - Execute statements from file and exit.
```

Comment:
really? Or does this have to be combined with --execute .. we should make an example section for this .. check out the stuff in the book .... pretty sure I wrote that up there